homeassistant/components/netgear_lte/__init__.py

```diff
@@ -212,33 +161,33 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     host = entry.data[CONF_HOST]
     password = entry.data[CONF_PASSWORD]
 
-    if DOMAIN not in hass.data:
+    if not (data := hass.data.setdefault(DOMAIN, {})):
```

Comment:
Why do we need this if statement?

Reply:
We only need one web session for the integration. Multiple entries can use the same one. Do you prefer it the old way?